doc/api/http2.md

```diff
@@ -1,9 +1,13 @@
 # HTTP/2
-
+<!-- YAML
+added: v8.4.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/22466
+    description: HTTP/2 has graduated from experimental status.
```

Comment:
Nit: Instead of "graduated from experimental status", it might be clearer to say "graduated to stable status" or even "graduated from experimental status to stable status".

Reply:
Going even further with the nit: Leave "status" out and either say nothing or "stability index". And leave out "graduated".

> HTTP/2 is now Stability Index 2 (Stable).

Or:

> HTTP/2 is now Stability Index 2 (Stable). Previously, it had been Stability Index 1 (Experimental).

And if you *really* are feeling motivated, find some way to link to https://nodejs.org/api/documentation.html#documentation_stability_index.

Reply:
Feel free to make those edits yourself if you'd like.

Reply:
Will do!